apps/web-giddh/src/app/vat-report/vat-report-filters/vat-report-filters.component.ts

```
@@ -210,6 +215,7 @@ export class VatReportFiltersComponent implements OnInit, OnChanges {
                 this.isConsolidatedBranch = response.isBranchConsolidated;
             }
         });
+        this.isTaxRestrictedModule = this.activeCompany?.subscription?.planDetails?.restrictedModules.hasOwnProperty(this.restrictedModules.TaxFilling);
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)